src/main/java/org/lateralgm/joshedit/JoshTextPanel.java

```diff
@@ -55,15 +55,19 @@ public JoshTextPanel() {
   public JoshTextPanel(String code) {
     this(Runner.splitLines(code));
   }
+  
+  public JoshTextPanel(String code, Font font) {
+    this(Runner.splitLines(code), font);
+  }
 
   public JoshTextPanel(String[] codeLines) {
```

Comment:
I see. This constructor didn't change, but you broke the delegation that was happening here when you edited that other constructor. Please fix that.

Reply:
Oops my bad: fc837534b2d7d6bd05189a87363fce1b97ca6615